ext/node/polyfills/process.ts

```diff
@@ -337,7 +339,20 @@ function uncaughtExceptionHandler(err: any, origin: string) {
   process.emit("uncaughtException", err, origin);
 }
 
-let execPath: string | null = null;
+export let execPath: string = Object.freeze({
+  __proto__: String.prototype,
+  toString() {
+    execPath = Deno.execPath();
```

Comment:
This could happen during the bootstrap process when we're not warming up? Then we won't have to call it each time `execPath` is accessed?